tests/features/bootstrap/lightning.behat.inc

```diff
@@ -401,7 +401,12 @@ class LightningSubContext extends DrupalSubContextBase {
    * @BeforeScenario
    */
   public function setFilePath() {
-    $this->minkContext->setMinkParameter('files_path', __DIR__ . '/../../files');
+
+    $existing = $this->minkContext->getMinkParameter('files_path');
+    if (empty($existing)) {
+
```

Comment:
Nitpick, but I can't help myself -- can the extra line after empty($existing) be removed? Once that's done, this is good to merge.